DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/loop_carried_dependency/sample.json

```diff
@@ -34,19 +38,23 @@
       {
         "id": "fpga_emu",
         "steps": [
+          "dpcpp --version",
+          "cd ../../..",
```

Comment:
this is Windows, you have to use \ not /
(comment applies in multiple places)

Reply:
that's what I had initially, but thought it doesn't matter. Also, line 45 "cmake -G \"NMake Makefiles\"..." has the forward slash in context of the literal quote character. Are you sure we should proceed with \ instead of /?

Reply:
Not 100% sure - I guess we'll try it and see

Reply:
The CI passed, so you were right again Yili, good work!